alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/Destination.kt

```diff
@@ -68,13 +71,26 @@ data class Destination(
         return toXContent(builder, ToXContent.EMPTY_PARAMS)
     }
 
+    private fun XContentBuilder.optionalUserField(user: User?): XContentBuilder {
```

Comment:
nit: Duplication of one in Monitor, just move to ElasticExtensions and use in both